mainapp/views.py

```diff
@@ -239,3 +240,39 @@ def clubsandcouncils():
             cou['clubs'].append(club)
             response['councils'].append(cou)
     return response
+
+@csrf_exempt
+def importantcontacts(request):
+    response = {}
+    response['status'] = 0
+    if request.method == 'POST':
+        post = json.loads(request.body)  # request.POST
+        role = post['role']
+        contacts = ImpContact.objects.filter(role_type=role)
+        if contacts:
+            response['data'] = [
+                {
+                    "name" : contact.name,
+                    "email" : contact.email,
+                    "phone" : contact.phone,
+                } for contact in contacts ]
+            return JsonResponse(response)
```

Comment:
Why do we need to send role. Keep this end point like -- if you recieve a request just send all the important contacts. Don't keep it protected behind any verification.